experiments/azure/cyclecloud/cpu/experiment/README.md

```diff
@@ -87,125 +87,143 @@ rm singularity-ce*.tar.gz
 
 Batch script:
 
-```console
+```bash
+size=128
 oras login ghcr.io --username vsoch
-srun --time=00:04 -N 2 slurm-single-node-benchmarks.sh
+srun --time=00:04 -N ${size} slurm-single-node-benchmarks.sh
```

Comment:
All of this repetition (across environments) had me thinking today if there could have been a more programmatic way to handle it all. I like to have the bash scripts right in front of me, but even something simple with templates that would "sploot" out a matrix across sizes, and run some number of setup commands, would have been hugely useful.

Reply:
Yeah, I was thinking the same, but probably too late for this set of experiments.

Reply:
Agree.